src/Modal/index.js

```diff
@@ -241,6 +249,10 @@ class Modal extends React.PureComponent<Props, State> {
     if (ev.target instanceof HTMLDivElement) {
       this.setState({
         scrolled: ev.target.scrollTop >= this.offset,
+        fixedClose: ev.target.scrollTop >= 1,
+        fullyScrolled:
+          this.props.fixedFooter &&
+          ev.target.scrollTop >= ev.target.scrollHeight - ev.target.clientHeight - 10,
```

Comment:
what is `-10` ?

Reply:
Ok, so please can you add some comments to code ?

Reply:
OK 👌